### PR TITLE
Changes In Events and Teams

### PR DESCRIPTION
**Pull request by Manas1820**
# Description

Please include the summary of the change as well as the issue that is fixed (if applicable). Also if any dependencies that are required for this change to work.

Fixes # (issue)

## Type of change

Please choose the relavant type of change.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Also summarise the process for us to reproduce.

- [x] Test A: checked through my emulator.
- [ ] Test B: Firebase testing labs
- [ ] Test C: Test through flutter test code
- [ ] Test D: Other (Please specify)

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this application
- [x] I have checked the code working on my emulator
- [x] I have commented my code for better understanding
- [x] These changes generate no new warnings
- [x] These changes do not effect the existing functionalities,

### CODE REVIEW

**Review by hayat-tamboli — COMMENTED** [review]
@Manas1820  what did you try  to do in add post page

**Comment by Manas1820** [comment]
just passed a status variable with an empty string and similarly for the type of the event

File: lib/view/addPost.dart [hunk 1]
```diff
@@ -84,52 +81,124 @@ class _AddPostState extends State<AddPost> {
...
-      divisions: 25,
-    );
     return Scaffold(
-        // body: NestedScrollView(
-        // headerSliverBuilder: addPostSliverAppBar,
-        appBar: AppBar(
-          leading: BackButton(),
-          title: buildTitle(context, "Add Post"),
-        ),
-        body: PageView(
```
Comment (hayat-tamboli) [inline lib/view/addPost.dart:111]: Why did you remove page View

**Comment by Manas1820** [comment]
wait I was having conflicts that  why I removed your code

**Review by hayat-tamboli — APPROVED** [review]
OK I will have to that again, I will approve it
